csrankings-i.csv

```diff
@@ -108,6 +108,7 @@ Igor A. Semaev,University of Bergen,https://www.uib.no/en/persons/Igor.A..Semaev
 Igor Carboni Oliveira,University of Warwick,https://warwick.ac.uk/fac/sci/dcs/people/igor_carboni_oliveira,aaGW2qwAAAAJ
 Igor Fabio Steinmacher,Northern Arizona University,http://www.igor.pro.br,I8o8rfoAAAAJ
 Igor Goryanin,University of Edinburgh,https://www.inf.ed.ac.uk/people/staff/Igor_Goryanin.html,E-duT6oAAAAJ
+Igor Gilitschenski,University of Toronto,https://www.gilitschenski.org/igor/,Nuw1Y4oAAAAJ

```

Comment:
Please verify >= 75% academic year appt. at UofT.

Reply:
Igor is 100% at UofT. + 20% at TRI (@igilitschenski)

https://www.utm.utoronto.ca/math-cs-stats/people/igor-gilitschenski 
https://web.cs.toronto.edu/people/faculty-directory

We have multiple campuses with the same graduate department.

Reply:
We all have main campus appointments for graduate studies -- which is the purpose of this ranking. 

The university is too large to house 90k UG students in one place. Hence smaller centers exist in the GTA area to spread the class load. It is much like colleges in Oxford. All members still count as Oxford faculty, but with hierarchical affiliation schema.

Many of the CS folks at Toronto such as Daniel Wigdor, Sanja Fidler, myself, Igor G. Florian Shkruti, Jessica Burgner Kahrs, have simultaneous appointments to support UG teaching. 
https://www.utm.utoronto.ca/math-cs-stats/people
https://www.utsc.utoronto.ca/cms/faculty

Notably, these are not separate graduate units and have no ability to admit graduate students. There is a single process to admit graduate students and award graduate degrees centrally managed at the university of toronto level.

Reply:
@emeryberger it would be great to have your opinion on this and if you agree a merge to the page. It would help since students are in decision making phase 😅

Reply:
@emeryberger this is a kind reminder to add Igor to the faculty list at CS